prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -116,19 +130,23 @@ class FakeReport(val metadata: Metadata) {
                     else -> TODO()
                 }
             }
-            Element.Type.TELEPHONE -> faker.numerify("2#########:1:") // faker.phoneNumber().cellPhone()
```

Comment:
The problem was that our phone number Translator was breaking occasionally on FakeReport generated numbers with leading zeros or ones.   The `2` was a hack fix for that.  Have we fixed this in some better way?